ui/lib/pki/addon/routes/configuration.js

```diff
@@ -4,5 +4,34 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { hash } from 'rsvp';
 
-export default class PkiConfigurationRoute extends Route {}
+export default class PkiConfigurationRoute extends Route {
+  @service store;
+
+  async fetchUrls(backend) {
+    try {
+      return await this.store.findRecord('pki/urls', backend);
+    } catch (e) {
+      return e.httpStatus;
+    }
+  }
+
+  async fetchCrl(backend) {
+    try {
+      return await this.store.findRecord('pki/crl', backend);
+    } catch (e) {
+      return e.httpStatus;
+    }
+  }
+
+  model() {
+    const engine = this.modelFor('application');
+    return hash({
+      engine,
+      urls: this.fetchUrls(engine.id),
+      crl: this.fetchCrl(engine.id),
```

Comment:
This is all good but just a suggestion to simplify, you could return the promise with a catch. For example:
```js
urls: this.store.findRecord('pki/crl', engine.id).catch((e) => e.httpStatus)
```